12/umbraco-commerce/SUMMARY.md

```diff
@@ -56,4 +56,16 @@
 
 ## Reference
 
+* [Storefront API](reference/storefront-api/README.md)
+  * [Endpoints](reference/storefront-api/endpoints/README.md)
+    * [Order](reference/storefront-api/endpoints/order.md)
+    * [Checkout](reference/storefront-api/endpoints/checkout.md)
+    * [Product](reference/storefront-api/endpoints/product.md)
+    * [Customer](reference/storefront-api/endpoints/customer.md)
+    * [Store](reference/storefront-api/endpoints/store.md)
+    * [Currency](reference/storefront-api/endpoints/currency.md)
+    * [Country](reference/storefront-api/endpoints/country.md)
+    * [Payment method](reference/storefront-api/endpoints/payment-method.md)
+    * [Shipping method](reference/storefront-api/endpoints/shipping-method.md)
+    * [Content](reference/storefront-api/endpoints/content.md)
```

Comment:
@mattbrailsford - I've updated the SUMMARY file now.

Another "funny" think with GitBook is that it wants to sync the link titles from the SUMMARY file with the titles of the articles they are referencing.
Currently, you've given the endpoint articles titles like `Storefront xxx API Endpoints`, but I've added only the `xxx` part to the SUMMARY file 😅 

I think that the entire title you've added is a little too long for the navigation, and would make it look a little cluttered.
How about we shorten the titles to something like `xxx Endpoints` instead?
We're already in the context of the Storefront API docs, so to me, it would be OK to leave those out.
What do you think?

Reply:
Yea, I'm cool with that 👍

Reply:
I never felt happy with them anyway to be honest